core/rip7560_abi_constants.go

```diff
@@ -0,0 +1,75 @@
+package core
+
+const AcceptAccountMethodSig = uint64(0x1256ebd1)   // acceptAccount(uint256,uint256)
+const AcceptPaymasterMethodSig = uint64(0x03be8439) // acceptPaymaster(uint256,uint256,bytes)
+const SigFailAccountMethodSig = uint64(0x7715fac2)  // sigFailAccount(uint256,uint256)
+const PaymasterMaxContextSize = 65536
+const Rip7560AbiVersion = 0
+
+const ValidateTransactionAbi = `
```

Comment:
nitpic: could create an "interface" abi, with multiple functions (not critical, though it would make also the methodsigs calculated by the encoder/decoder lib)

Reply:
We should be generating this file, so I don't think we need to spend time polishing the manually created one for now.